javascript/javascript1/week3/homework-week3.js

```diff
@@ -0,0 +1,166 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+ 
+  const indexToRemove = names.indexOf(nameToRemove);
+  
+  if (indexToRemove !== -1) {
+    names.splice(indexToRemove, 1);
+  }
+  
+  console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
```

Comment:
This is good, but try to use function to do the same logic. In real life we won't write the code in global scope, but inside functions. Since these exercises are a practise for you try to use function